guides/rules/no-broken-super-chain.md

```diff
@@ -0,0 +1,25 @@
+# Don't break the super chain
+
+## Rule name: `no-broken-super-chain`
+
+If you are overriding the `init` lifecycle hook in Ember classes like Component, Mixin, etc. it is necessary that you include a call to `_super`.
+
+```javascript
+// GOOD
+
+export default Ember.Component.extend({
+  init() {
+    this._super(...arguments);
```

Comment:
init doesn't have any arguments, so this causes two unnecessary array allocations as well as adds 4 additional function calls to the overhead.